### PR TITLE
LibCore+LibGUI: add StandardPaths::documents_directory and add Documents and Downloads directories to tray shortcuts in FilePicker

### PR DESCRIPTION
**Pull request by ZikiFlicky**

### CODE REVIEW

**Review by awesomekling — CHANGES_REQUESTED** [review]
This is not used by anything? Please don't add dead code. Instead, add this in a PR that uses it for something.

**Comment by ZikiFlicky** [comment]
I was planning to add a Documents shortcut in the left toolbar in the FilePicker widget that will use the method, but I wasn't sure if it belongs in this pull request. Sorry for not mentioning it 😬

**Comment by ZikiFlicky** [comment]
I added Documents and Downloads folders to FilePicker. can you review the changes?

File: Userland/Libraries/LibGUI/FilePicker.cpp [hunk 1]
```diff
@@ -243,12 +243,18 @@ FilePicker::FilePicker(Window* parent_window, Mode mode, const StringView& file_
...
 
     m_model->on_complete = [&] {
         if (m_model->root_path() == Core::StandardPaths::home_directory()) {
             home_button.set_checked(true);
         } else if (m_model->root_path() == Core::StandardPaths::desktop_directory()) {
             desktop_button.set_checked(true);
+        } else if (m_model->root_path() == Core::StandardPaths::downloads_directory()) {
+            downloads_button.set_checked(true);
+        } else if (m_model->root_path() == Core::StandardPaths::documents_directory()) {
+            documents_button.set_checked(true);
```
Comment (Dexesttp) [inline Userland/Libraries/LibGUI/FilePicker.cpp:257]: You will need to uncheck these buttons when another path is selected, which is done right below :) Just add the `downloads_button.set_checked(false);` and `documents_button.set_checked(false);` to the else branch.

**Pull request closed without merging**